templates/webapi/branding/openSUSE/external_reporting.html.ep

```diff
@@ -36,6 +36,11 @@
     Leanos => 'Server',
     Installer => 'Server',
 ); %>
+<% my %public_sle_products = (
+    Server => 'Server',
+    Desktop => 'Desktop',
+    'High Availability Extension' => 'High Availability', # the public version leaves out the "Extension" suffix
+); %>
```

Comment:
You could also just prefix each line with a `% ` instead of wrapping them in `<% ... %>`. Works the same.